data/fields/sidewalk.json

```
@@ -0,0 +1,34 @@
+{
+    "key": "sidewalk:both",
+    "keys": [
+        "sidewalk:left",
+        "sidewalk:right"
+    ],
+    "reference": {
+        "key": "sidewalk"
+    },
+    "type": "directionalCombo",
+    "label": "Sidewalks",
+    "placeholder": "Yes, No, Mapped using separate ways",
+    "strings": {
+        "types": {
+            "sidewalk:left": "Left Side",
+            "sidewalk:right": "Right Side"
+        },
+        "options": {
+            "yes": {
+                "title": "Yes",
+                "description": "There is a sidewalk"
+            },
+            "no": {
+                "title": "No",
+                "description": "No sidewalk"
+            },
+            "separate": {
+                "title": "Sidewalk Mapped Separately",
+                "description": "The sidewalk are mapped using separate ways"
```

```suggestion
                "description": "The sidewalk is mapped using separate ways"
```

A minor grammar suggestion.